src/dataflow/src/render/join/linear_join.rs

```diff
@@ -299,6 +301,10 @@ where
         closure: JoinClosure,
         errors: &mut Vec<Collection<G, DataflowError>>,
     ) -> Collection<G, Row> {
+        println!(
+            "differential join stream/lookup key: {:?}\t{:?}",
+            stream_key, lookup_key
+        );
```

Comment:
Probably to delete?

Reply:
Argh, that was my attempt at making progress for #6672 :-D. I'll get it out of here.